ui/segment.reel/segment.css

```diff
@@ -0,0 +1,12 @@
+.montage-Segment {
+    /* to vertical align text labels */
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+
+    -webkit-box-align: center;
+    -webkit-align-items: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
```

Comment:
Sounds good. Note that this is not about what our developers will do/use (Studio or mop could indeed handle that), but about the fact that we're responsible for the code we commit, regardless of how that code was written or generated. With a comment, this code will be self-explanatory and the -webkit-box will be fine.